review: 01 getting started

topless opened the pull request:
- cleaned up wording
- updated invenio cookie cutted version
- MD045 alt text for images
- MD014 dollar-signs-used-before-commands-without-showing-output (easier command copy)
- child of #50

diegodelemos commented on 01-getting-started/README.md:
```diff
@@ -25,51 +24,49 @@ use: Ctrl+Shift+V (paste), Ctrl+Shift+C (copy), Ctrl+Shift+X (cut).
-as an execption. Afterwards, you should now see your first Invenio instance
-running:
+Simply bypass this warning, by clicking "Advanced" and confirm the certificate as an exception.
+Afterwards, you should now see your first Invenio instance running:
```
Minor
```suggestion
Afterwards, you should see your first Invenio instance running:
```

topless committed:
review: 01 getting started

- cleaned up wording
- updated invenio cookie cutted version
- MD045 alt text for images
- MD014 dollar-signs-used-before-commands-without-showing-output (easier command copy)
- child of #50
